toolchain/eslint-config/package.json

```diff
@@ -33,7 +32,6 @@
     "svelte-eslint-parser": "0.43.0"
   },
   "devDependencies": {
-    "@eslint/compat": "1.2.4",
```

Comment:
eslint-plugin-deprecation was the last rule to need this dependency. All plugins are now compatible with eslint 9.x.